client/pages/config/email.vue

```diff
@@ -146,6 +151,29 @@ export default {
         ...this.settings
       }
     },
+    async loadUsers() {
+      this.users = await this.$axios
+        .$get('/api/users')
+        .then((res) => {
+          return res.users.sort((a, b) => {
+            return a.createdAt - b.createdAt
+          })
+        })
+        .catch((error) => {
+          console.error('Failed', error)
+          return []
+        })
+    },
+    getAccessibleBy(device) {
+      const user = device.availabilityOption
+      if (user === 'userOrUp') return 'Users (excluding Guests)'
+      if (user === 'guestOrUp') return 'Users (including Guests)'
+      if (user === 'specificUsers') {
+        this.loadUsers()
```

Comment:
For future ref, calling loadUsers here could result in multiple simultaneous calls.